pkg/containertools/dockerfilegenerator.go

```diff
@@ -46,9 +49,17 @@ func (g *IndexDockerfileGenerator) GenerateIndexDockerfile(binarySourceImage, da
 
 	// Labels
 	dockerfile += fmt.Sprintf("LABEL %s=%s\n", DbLocationLabel, DefaultDbLocation)
+	dockerfile += fmt.Sprintf("LABEL %s=%s\n", ConfigsLocationLabel, DefaultConfigFolderLocation)
 
 	// Content
-	dockerfile += fmt.Sprintf("ADD %s %s\n", databasePath, DefaultDbLocation)
```

Comment:
I like how flexible the new method is, but it also drops the guarantee that the `databasePath` is placed at `DefaultDbLocation`.

It is now less clear which of the inputs is used as the database, and the onus is on the method client to know to add a mapping to `DefaultDbLocation`.

Reply:
@njhale not sure how we feel about documenting that in the method doc to make it more clear? It also does feel like it might be overkill when only one type of stored format (sqlilite OR configs OR anything else) is being used though. Do we feel like reverting to the previous design that extended the number of method arguments (which was slightly unclean but simple to read) is not favorable over ensuring future extensibility of this method?   
It should be noted again that extending the number of method argument will only be a stop gap measure, till the feature for migrating old indexes to new format is implement, in which case we'll be back to only one format. 

The other option is leave the method untouched, so this PR would create the configs, but not load them to the container image. And when the migration/serve is ready, a new PR would switch the data being loaded to configs instead of the database.